src/Log/IsoLog.js

```diff
@@ -294,7 +298,7 @@ module.exports = class Log {
 			const now = this.getDatetimeString()
 
 			const thingType = typeof thingToLog
-			this.getLine()
+			return this.getLine()
```

Comment:
Anywhere that is async now returns a promise. This ensures every method is independently testable without timeouts, which is fragile by comparison.

The implementer of the class can decide to wait until the logging is done, which may make sense with async transports that are critical.

But, really, this is more for ensuring good principles in that every method is independently testable.